apps/protoc-gen-openapi/plugin_test.go

```diff
@@ -65,3 +65,26 @@ func TestBodyMappingOpenAPI(t *testing.T) {
 	// if the test succeeded, clean up
 	os.Remove("openapi.yaml")
 }
+
+func TestMapFieldsOpenAPI(t *testing.T) {
+	var err error
+	// Run protoc and the protoc-gen-openapi plugin to generate an OpenAPI spec.
+	err = exec.Command("protoc",
+		"-I", "../../",
+		"-I", "../../third_party",
+		"-I", "examples",
+		"examples/tests/mapfields/message.proto",
+		"--openapi_out=.").Run()
+	if err != nil {
+		t.Logf("protoc failed: %+v", err)
```

Comment:
Nit: `t.Fatalf` is equivalent to `t.Logf` followed by `t.FailNow`. Both occurrences can be updated.

Reply:
Thanks! I've updated all six instances in that test.